concordium-node/src/configuration.rs

```diff
@@ -158,109 +188,146 @@ pub struct BakerConfig {
+        default_value = "none",
+        env = "CONCORDIUM_NODE_BAKER_HEAP_PROFILING"
     )]
     pub heap_profiling: String,
     #[cfg(feature = "profiling")]
-    #[structopt(long = "time-profiling", help = "Profile the time in the Haskell subsystem")]
+    #[structopt(
+        long = "time-profiling",
+        help = "Profile the time in the Haskell subsystem",
+        env = "CONCORDIUM_NODE_BAKER_TIME_PROFILING"
+    )]
     pub time_profiling: bool,
     #[cfg(feature = "profiling")]
     #[structopt(
         long = "backtraces",
-        help = "Show bactraces generated by exceptions in the Haskell subsystem"
+        help = "Show bactraces generated by exceptions in the Haskell subsystem",
+        env = "CONCORDIUM_NODE_BAKER_BACKTRACES"
     )]
     pub backtraces_profiling: bool,
     #[cfg(feature = "profiling")]
     #[structopt(
         long = "stack-profiling",
         help = "Include memory occupied by threads in the heap profile. Only has effect if \
-                `heap-profiling` is enabled."
+                `heap-profiling` is enabled.",
+        env = "CONCORDIUM_NODE_BAKER_STACK_PROFILING"
     )]
     pub stack_profiling: bool,
     #[cfg(feature = "profiling")]
     #[structopt(
         long = "profiling-sampling-interval",
         help = "Profile sampling interval in seconds",
-        default_value = "0.1"
+        default_value = "0.1",
+        env = "CONCORDIUM_NODE_BAKER_PROFILING_SAMPLING_INTERVAL"
     )]
     pub profiling_sampling_interval: String,
-    #[structopt(long = "haskell-gc-logging", help = "Enable Haskell garbage collection logging")]
+    #[structopt(
+        long = "haskell-gc-logging",
+        help = "Enable Haskell garbage collection logging",
+        env = "CONCORDIUM_NODE_BAKER_HASKELL_GC_LOGGING"
+    )]
     pub gc_logging: Option<String>,
     #[structopt(
         long = "haskell-rts-flags",
         help = "Haskell RTS flags to pass to consensus.",
-        default_value = ""
+        default_value = "",
+        env = "CONCORDIUM_NODE_BAKER_HASKELL_RTS_FLAGS"
     )]
     pub rts_flags: Vec<String>,
     #[structopt(
         long = "maximum-block-size",
         help = "Maximum block size in bytes",
-        default_value = "4194304"
+        default_value = "4194304",
+        env = "CONCORDIUM_NODE_BAKER_MAXIMUM_BLOCK_SIZE"
     )]
     pub maximum_block_size: u32,
     #[structopt(
         long = "block-construction-timeout",
         help = "Block construction timeout in milliseconds",
-        default_value = "3000"
+        default_value = "3000",
+        env = "CONCORDIUM_NODE_BAKER_BLOCK_CONSTRUCTION_TIMEOUT"
     )]
     pub block_construction_timeout: u32,
     #[structopt(
         long = "transaction-insertions-before-purge",
         help = "Number of transaction insertions between purges on the transaction table",
-        default_value = "1000"
+        default_value = "1000",
+        env = "CONCORDIUM_NODE_BAKER_TRANSACTION_INSERTIONS_BEFORE_PURGE"
```

Comment:
This would be more appropriately in the CONSENSUS grouping, i.e., 

`CONCORDIUM_NODE_CONSENSUS_...`